scripts/slack_notifications.py

```diff
@@ -44,6 +44,12 @@ def send_notification():
         failure = nightly_slack_messages[message_key]
         message = {"text": ":blobonfire: Nightly build failed %s" % failure}
```

Comment:
nit: not quite this PR, but I think we use `f"{some_variable}"`-style format strings more commonly throughout the codebase